src/hooks/useWs/index.tsx

```diff
@@ -19,10 +19,20 @@ const useWS = <T extends TSocketEndpointNames>(name: T) => {
   }, []);
 
   const send = useCallback(
-    async (data?: Parameters<typeof apiManager.augmentedSend<T>>[1]) => {
+    async (data?: Parameters<typeof apiManager.augmentedSend<T>>[0]) => {
+      let payload = data;
+      console.log(payload);
+
+      if (name) {
+        payload = { [name]: 1, ...payload };
+      } else {
```

Comment:
Isn't it adds both payloads together?
How does this handle private API calls like "trading_platform_available_accounts"?

Reply:
@matin-deriv for a private api call we don't have name param so it will just spread the request response as the payload and execute the call, since the name is available only in the dropdown or in places where you use the hook inside the codebase